decidim-core/spec/system/account_spec.rb

```diff
@@ -123,20 +123,56 @@
           expect(user.reload.valid_password?("sekritpass123")).to eq(false)
         end
       end
+    end
 
-      context "when updating the email" do
-        it "needs to confirm it" do
-          within "form.edit_user" do
-            fill_in :user_email, with: "foo@bar.com"
+    context "when updating the email" do
+      let(:pending_email) { "foo@bar.com" }
 
-            find("*[type=submit]").click
-          end
+      before do
+        within "form.edit_user" do
+          fill_in :user_email, with: pending_email
 
-          within_flash_messages do
-            expect(page).to have_content("email to confirm")
-          end
+          perform_enqueued_jobs { find("*[type=submit]").click }
+        end
+
+        within_flash_messages do
+          expect(page).to have_content("You'll receive an email to confirm your new email address")
         end
       end
+
+      after do
+        clear_enqueued_jobs
+      end
+
+      it "tells user to confirm new email" do
+        expect(page).to have_content("Email change verification")
+        expect(page).to have_selector("#user_email[disabled='disabled']")
+        expect(page).to have_content("We've sent an email to #{pending_email} to verify your new email address")
+      end
+
+      it "resend confirmation" do
+        within "#email-change-pending" do
+          click_link "Send again"
+        end
+        expect(page).to have_content("Confirmation email resent successfully to #{pending_email}")
+        perform_enqueued_jobs
```

Comment:
It's not working if you wrap the `click_link` in a `perform_enqueued_jobs` block?

Why do we need to call it twice?

Reply:
Yeah this is only way I got it working..

Reply:
OK, I also have similar problems sometimes with the development environment, so I think we can just let it be as is.